src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -0,0 +1,144 @@
+yet.
+
+For status updates, see [flutter#146957].
+
+[flutter#146957]: https://github.com/flutter/flutter/issues/146957
+
+## How to add Swift Package Manager integration to a project manually
+
+{% tabs %}
+{% tab "iOS project" %}
+
+{% include docs/swift-package-manager/migrate-ios-project-manually.md %}
+
+{% endtab %}
+{% tab "macOS project" %}
+
+{% include docs/swift-package-manager/migrate-macos-project-manually.md %}
+
+{% endtab %}
+{% endtabs %}
+
+## How to use a Swift Package Manager Flutter plugin that requires a higher OS version
+
+If a Swift Package Flutter Manager plugin requires a higher OS version than
+the project, you might get an error like this:
+
+```plaintext
+Target Integrity (Xcode): The package product 'plugin_name_ios' requires minimum platform version 14.0 for the iOS platform, but this target supports 12.0
+```
+
+To use the plugin, you'll need to increase the **Minimum Deployments** of your
+app's target.
+
+:::warning
+This raises your app's minimum OS version requirement.
+:::
+
+{% render docs/captioned-image.liquid,
+image:"development/packages-and-plugins/swift-package-manager/minimum-deployments.png",
+caption:"The target's **Minimum Deployments** setting" %}
+
+## How to add Swift Package Manager integration to a custom Xcode target
+
+Your Flutter Xcode project can have custom [Xcode targets][] to build additional
+products, like frameworks or unit tests.
+You can add Swift Package Manager integration to these custom Xcode targets
+as well.
+
+Follow the steps in
+[How to add Swift Package Manager integration to a project manually][manualIntegration].
+
+In [Step 1][manualIntegrationStep1], list item 6 use your custom target instead
+of the `Flutter` target.
+
+In [Step 2][manualIntegrationStep2], list item 6 use your custom target instead
+of the `Flutter` target.
+
+[Xcode targets]: https://developer.apple.com/documentation/xcode/configuring-a-new-target-in-your-project
+[manualIntegration]: /packages-and-plugins/swift-package-manager/for-app-developers/#how-to-add-swift-package-manager-integration-to-a-project-manually
+[manualIntegrationStep1]: /packages-and-plugins/swift-package-manager/for-app-developers/#step-1-add-fluttergeneratedpluginswiftpackage-package-dependency
+[manualIntegrationStep2]: /packages-and-plugins/swift-package-manager/for-app-developers/#step-2-add-run-prepare-flutter-framework-script-pre-action
+
+## How to remove Swift Package Manager integration
+
+The Flutter CLI migrates your project to add Swift Package Manager integration.
+This makes your project download the Swift packages that your Flutter plugins
+depend on.
+
+To undo this migration:
+
+1. [Turn off Swift Package Manager][].
+
+2. Open your app (`ios/Runner.xcworkspace` or `ios/Runner.xcworkspace`) in
+   Xcode.
+
+3. Navigate to **Package Dependencies** for the project.
+
+4. Click on the `FlutterGeneratedPluginSwiftPackage` package and then click the
+   **-** button.
```

Comment:
**issue:** Use the button's label per [GSG](https://developers.google.com/style/ui-elements#buttons).

```suggestion
4. Click the `FlutterGeneratedPluginSwiftPackage` package,
    then click <span class="material-symbols-outlined">add</span>.
```

Reply:
Updated. FYI the `-` appears too high:
 
![image](https://github.com/user-attachments/assets/52e332ab-4e42-4bea-b808-c3551f4fa1f5)

It looks like I need to add a `vertical-align: text-bottom` to the `span`. 

![image](https://github.com/user-attachments/assets/95b86de4-3470-4be6-8b90-074ce93e6644)

Should I add that globally on the website for all `p span.material-symbols-outlined`?